ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -703,12 +702,14 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
         ).subscribe();
     }
 
-
     private meetsActivityRequirements(currentAddress: Address | null): boolean {
         if (this.block.requireVerified && !currentAddress) {
             return false;
         }
-        return !(this.block.requirePhone && currentAddress && !(currentAddress.phone));
+        if (this.block.requirePhone && !(currentAddress?.phone)) {
+            return false;
+        }
+        return this.enoughDataToSave(currentAddress);
     }
```

Comment:
After reading the code more carefully and having established that "requireVerified" really just means "required":
```typescript    
private meetsActivityRequirements(currentAddress: Address | null): boolean {
        if (this.block.requireVerified && !(this.enoughDataToSave(currentAddress)) {
            return false;
        } else {
           return  !(this.block.requirePhone && !(currentAddress?.phone));
    }
```

(enoughDataToSave already checks to see if address is null)

Reply:
Thank you! 
Fixed in 7f6464471fd570d05b05e52698ebe5b0fc663c54